src/utils/isDecimal.js

```diff
@@ -0,0 +1,7 @@
+export default number => {
```

Comment:
We can improve this by 

`export default () => typeof number !== 'string' && number.toString().indexOf('.') !== -1`

Reply:
Done thanks!